app_wallabag2_fr.md

```diff
@@ -0,0 +1,30 @@
+## <img src="https://wallabag.org/user/themes/boxify/img/logo-wallabag-black.svg">
```

Comment:
I will use the one you mentioned, no need to duplicate the space for almost the same logo.
(I corrected the English version too)